errors/daemon.go

```diff
@@ -372,7 +372,7 @@ var (
 	// volume specification isn't valid.
 	ErrorCodeVolumeInvalid = errcode.Register(errGroup, errcode.ErrorDescriptor{
 		Value:          "VOLUMEINVALID",
-		Message:        "Invalid volume specification: %s",
+		Message:        "Invalid volume specification: '%s'",
```

Comment:
do you think it would be wise to add a comment, explaining that this should *not* be changed to `%q`? I can foresee someone "optimizing" it in the future (same for the others below)